src/demo-app/badge/badge-demo.ts

```diff
@@ -0,0 +1,25 @@
+import {Component} from '@angular/core';
+import {DomSanitizer} from '@angular/platform-browser';
+import {MatIconRegistry} from '@angular/material';
+
+@Component({
+  moduleId: module.id,
+  selector: 'badge-demo',
+  templateUrl: 'badge-demo.html',
+  styleUrls: ['badge-demo.css'],
+})
+export class BadgeDemo {
+
+  badgeContent = '1';
+  svgIcon = 'cat';
+
+  constructor(_iconRegistry: MatIconRegistry, sanitizer: DomSanitizer) {
+    _iconRegistry.addSvgIcon('home',
+      sanitizer.bypassSecurityTrustResourceUrl(
+        'https://upload.wikimedia.org/wikipedia/commons/3/34/Home-icon.svg'));
```

Comment:
You can use the home icon from the icon demo instead of fetching one remotely.

Reply:
I tried that, however, it was not working. That demo page is broken too if you open it up.